src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -76,7 +76,10 @@ class SideNav extends Component {
     const { navItemsLookup } = this.props;
 
     // Derive the parent-most nav item. This is O(n), which isn't great but I'm assuming there won't be 1000s of side nav items.
-    const parentMostNavItem = find(navItemsLookup, item => !item.parentID);
+    const parentMostNavItem = find(
+      navItemsLookup,
+      item => !item.parentID && includes(window.location.pathname, item.href),
+    );
```

Comment:
Would this make sense as filter prior to passing the data to the side-nav? I don't think it's unimaginable that another VA subdomain will eventually be rendered into a side nav

Reply:
Good idea! :)